app/src/main/java/ch/epfl/sdp/blindly/database/UserRepository.kt

```diff
@@ -50,30 +51,6 @@ class UserRepository @Inject constructor(
         return refreshUser(uid)
     }
 
-    /**
```

Comment:
Quite generally for this file we will have to be careful when merging with #222

Reply:
Yes it going to be a bit of a minefield...